doc/examples/perception_pipeline/perception_pipeline_tutorial.rst

```diff
@@ -1,198 +1,99 @@
-     padding_offset: 0.03
-     max_update_rate: 1.0
-     filtered_cloud_topic: filtered_cloud
-
-**The general parameters are:**
-
-* *sensor_plugin*: The name of the plugin that we are using.
-* *max_update_rate*: The octomap representation will be updated at rate less than or equal to this value.
-
-**Parameters specific to the Depth Map updater are:**
-
-* *image_topic*: This specifies the topic to listen on for a depth image.
-
-* *queue_size*: The number of images to queue up.
-
-* *near_clipping_plane_distance*: The minimum distance before lack of visibility.
-
-* *far_clipping_plane_distance*: The maximum distance before lack of visibility.
-
-* *shadow_threshold*: The minimum brightness of the shadow map below an entity for its dynamic shadow to be visible
-
-* *padding_offset*: The size of the padding (in cm).
-
-* *padding_scale*: The scale of the padding.
-
-* *filtered_cloud_topic*: The topic on which the filtered cloud will be published (mainly for debugging). The filtering cloud is the resultant cloud after self-filtering has been performed.
-
-
-Update the launch file
-++++++++++++++++++++++
-
-Add the YAML file to the launch script
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-You will now need to update the *sensor_manager.launch* file in the "launch" directory of your panda_moveit_config directory with this sensor information (this file is auto-generated by the Setup Assistant but is empty). You will need to add the following line into that file to configure the set of sensor sources for MoveIt to use: ::
-
- <rosparam command="load" file="$(find panda_moveit_config)/config/sensors_kinect_pointcloud.yaml" />
-
-If you are using depthmap change the name of the yaml file to ``sensors_kinect_depthmap.yaml``.
-Note that you will need to input the path to the right file you have created above.
-
-Octomap Configuration
-^^^^^^^^^^^^^^^^^^^^^
-You will also need to configure the `Octomap <http://octomap.github.io/>`_ by adding the following lines into the *sensor_manager.launch*: ::
+Even though you haven't completed :doc:`Getting Started </doc/tutorials/getting_started/getting_started>` ever, you can still run this tutorial's demo. Whereas, it is suggested to start with steps in :doc:`Getting Started </doc/tutorials/getting_started/getting_started>` for understanding what's going on in this tutorial.
 
- <param name="octomap_frame" type="string" value="odom_combined" />
- <param name="octomap_resolution" type="double" value="0.05" />
- <param name="max_range" type="double" value="5.0" />
+Get The Necessary 3D Pointcloud Data For Octomap Creation
+---------------------------------------------------------
+In this tutorial, you can use `previously recorded 3D pointcloud data <https://drive.google.com/file/d/1fPtDAtJKIiw2gpFOOwA2TrPZOfFU053W/view?usp=sharing>`_ or you can record your own bag file. For recording bag, firstly, it can be both run ``depth_camera_envrionment.launch.py`` and recorded bag using following commands.
```

Comment:
There's a typo in `depth_camera_envrionment.launch.py` and I don't understand the second sentence. Do you mind rewording it?

Reply:
You're absolutely right. I reworded like this 

```
For recording bag, firstly, we can run ``depth_camera_envrionment.launch.py`` file and then record the bag using following commands.
```